Add content/es/docs/concepts/configuration/pod-overhead.md (#17821)
kubernetes/website: content/es/docs/concepts/configuration/pod-overhead.md

---
reviewers:
- dchen1107
- egernst
- tallclair
title: Sobrecarga de Pod
content_template: templates/concept
weight: 20
---

{{% capture overview %}}

{{< feature-state for_k8s_version="v1.16" state="alpha" >}}

> Cuando se está ejecutando un Pod en un Nodo, el Pod por si mismo toma una cantidad de recursos del sistema. Estos recursos son adicionales a los recursos necesarios para hacer funcionar el/los contenedor(es) dentro del Pod.

Reviewer: ```suggestion
Cuando se está ejecutando un {{< glossary_tooltip text="Pod" term_id="pod" >}} en un {{< glossary_tooltip text="Nodo" term_id="node" >}}, el {{< glossary_tooltip text="Pod" term_id="pod" >}} por si mismo toma una cantidad de recursos del sistema. Estos recursos son adicionales a los recursos necesarios para hacer funcionar el/los contenedor(es) dentro del {{< glossary_tooltip text="Pod" term_id="pod" >}}.
```

> La _Sobrecarga de Pod_ es una característica para contabilizar los recursos consumidos por la infraestructura de pods que están arriba de las solicitudes y límites del/los contenedores.

Reviewer 2: Dado que Limits y Resquest son características de los Pods me parece que seria mejor no traducirlos y asi mantener el contexto
```suggestion
La _Sobrecarga de Pod_ es una característica para contabilizar los recursos consumidos por la infraestructura de Pods que están por encima de los valores de Requests y Limits del/los contenedor(es).
```

{{% /capture %}}

{{% capture body %}}

## Sobrecarga de Pod

> En Kubernetes, la sobrecarga de pod es configurado en el tiempo de [admisión](/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks) de acuerdo a la sobrecarga asociada con el [RuntimeClass](/docs/concepts/containers/runtime-class/) del pod.

Reviewer 2: ```suggestion
En Kubernetes, la sobrecarga de Pod es configurado en el tiempo de [admisión](/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks) de acuerdo a la sobrecarga asociada con el [RuntimeClass](/docs/concepts/containers/runtime-class/) del Pod.
```

> En Kubernetes, la sobrecarga de pod es configurado en el tiempo de [admisión](/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks) de acuerdo a la sobrecarga asociada con el [RuntimeClass](/docs/concepts/containers/runtime-class/) del pod.

Reviewer: A parte de los cambios sugeridos por @electrocucaracha, `la sobrecarga del Pod` es femenino, por lo que:

```suggestion
En Kubernetes, la sobrecarga del {{< glossary_tooltip text="Pod" term_id="pod" >}} es configurada en el tiempo de [admisión](/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks) de acuerdo a la sobrecarga asociada con el [RuntimeClass](/docs/concepts/containers/runtime-class/) del {{< glossary_tooltip text="Pod" term_id="pod" >}}.
```

> Cuando se habilita la opción de sobrecarga de pod, se considera la sobrecarga además de la suma de solicitudes de recursos del contenedor cuando se programa un pod. Del mismo modo, Kubelet incluirá la sobrecarga de pod cuando se dimensione el cgroup del pod, y cuando al realizar la clasificación de la expulsión de pods.

Reviewer 2: ```suggestion
Cuando se habilita la opción de sobrecarga de Pod, se considera la sobrecarga además de la suma de solicitudes de recursos del contenedor cuando se programa un Pod. Del mismo modo, Kubelet incluirá la sobrecarga de Pod cuando se dimensione el cgroup del Pod, y cuando al realizar la clasificación de la expulsión de Pods.
```

> Cuando se habilita la opción de sobrecarga de pod, se considera la sobrecarga además de la suma de solicitudes de recursos del contenedor cuando se programa un pod. Del mismo modo, Kubelet incluirá la sobrecarga de pod cuando se dimensione el cgroup del pod, y cuando al realizar la clasificación de la expulsión de pods.

Reviewer: ```suggestion
Cuando se habilita la opción de sobrecarga de {{< glossary_tooltip text="Pod" term_id="pod" >}}, se considera la sobrecarga además de la suma de solicitudes de recursos del contenedor cuando se programa un {{< glossary_tooltip text="Pod" term_id="pod" >}}. Del mismo modo, {{< glossary_tooltip text="Kubelet" term_id="kubelet" >}} incluirá la sobrecarga de {{< glossary_tooltip text="Pod" term_id="pod" >}} cuando se dimensione el  {{< glossary_tooltip text="cgroup" term_id="cgroup" >}}  del mismo, y también cuando se calcula el ranking de los {{< glossary_tooltip text="Pods" term_id="pod" >}} para expulsarlos de un nodo.
```

### Configuración

> Debe asegurarse de que la [puerta de características](/docs/reference/command-line-tools-reference/feature-gates/) `PodOverhead` esté activada (Está desactivada de manera predeterminada) en todo el cluster. Esto significa:

Reviewer 2: Me parece que es mejor no traducir *Feature Gates*
```suggestion
Debe asegurarse de que el [Feature Gates](/docs/reference/command-line-tools-reference/feature-gates/) `PodOverhead` esté activado (su valor está desactivado de manera predeterminada) en todo el clúster. Esto significa:
```

Reviewer: +1

- en {{< glossary_tooltip text="kube-scheduler" term_id="kube-scheduler" >}}
- en {{< glossary_tooltip text="kube-apiserver" term_id="kube-apiserver" >}}

> - en el {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} on each Node

Reviewer 2: ```suggestion
- en el {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} de cada nodo
```

> - en el {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} on each Node

Reviewer: ```suggestion
- en el {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} en cada {{< glossary_tooltip text="Pod" term_id="pod" >}} en un {{< glossary_tooltip text="Nodo" term_id="node" >}}.
```

> - en cualquier servidor de API personalizado que ocupe puertas de características.

Reviewer 2: ```suggestion
- en cualquier servidor de API personalizado que necesite Feature Gates
```

> - en cualquier servidor de API personalizado que ocupe puertas de características.

Reviewer: ```suggestion
- en cualquier servidor de API personalizado que necesite [Feature Gates](/docs/reference/command-line-tools-reference/feature-gates/).
```

{{< note >}}

> Los usuarios que pueden escribir recursos del tipo RuntimeClass pueden tener impacto en el rendimiento de la carga de trabajo en todo el cluster. Puede limitar el acceso ha esta habilidad usando los controles de acceso de Kubernetes.

Reviewer 2: ```suggestion
Los usuarios que pueden escribir recursos del tipo RuntimeClass son capaces de impactar el rendimiento de la carga de trabajo en todo el clúster. Puede limitar el acceso ha esta habilidad usando los controles de acceso de Kubernetes.
```

> Para obtener más detalles vea [Resumen de Autorización](/docs/reference/access-authn-authz/authorization/).

Reviewer: ```suggestion
Para obtener más detalles consulte [Resumen de Autorización](/docs/reference/access-authn-authz/authorization/).
```

{{< /note >}}

{{% /capture %}}

{{% capture whatsnext %}}

* [RuntimeClass](/docs/concepts/containers/runtime-class/)

> * [PodOverhead Design](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/20190226-pod-overhead.md)

Reviewer 2: ```suggestion
* [Diseño de sobrecarga de Pod](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/20190226-pod-overhead.md)
```
